modules/core/src/main/java/org/apache/ignite/internal/IgniteFeatures.java

```diff
@@ -146,7 +146,10 @@
     INVERSE_TCP_CONNECTION(35),
 
     /** Check secondary indexes inline size on join/by control utility request. */
-    CHECK_INDEX_INLINE_SIZES(36);
+    CHECK_INDEX_INLINE_SIZES(36),
+
+    /** Force rebuild, list or request indexes rebuild status from control script. */
+    INDEXES_MANIPULATIONS_FROM_CONTROL_SCRIPT(41);
```

Comment:
You forgot to reserve feature id on page https://ggsystems.atlassian.net/wiki/spaces/GG/pages/1192198276/Community+edition+features+list